src/UnderlineNav2/index.ts

```diff
@@ -1,10 +1,10 @@
 import {UnderlineNav as Nav, UnderlineNavProps} from './UnderlineNav'
-import {UnderlineNavLink, UnderlineNavLinkProps} from './UnderlineNavLink'
+import {UnderlineNavItem, UnderlineNavItemProps} from './UnderlineNavItem'
 
 const UnderlineNav = Object.assign(Nav, {
-  Link: UnderlineNavLink
+  Item: UnderlineNavItem
```

Comment:
This would be a breaking change wrt to the old UnderlineNav. We will need to mention it in the changelog.